File: cloud/dynamic-site/gt-cs-1331-py/app/gtcs1331/solutions/solution110.py
'''Solution 1.10 implementation'''
from gtcs1331.solution_api.solution import Solution

class Solution110(Solution):
    '''Solution 1.10'''
    id = "1.10"
    title = "Average Speed"

    def __answer(self):
        speed = 14.0 / 1.6 / (45.0 / 60.0 + 30.0 / 60.0 / 60.0)

        return [
            f"Average speed is {speed:.6f} km/h"]

    answer = property(__answer)

File: cloud/dynamic-site/gt-cs-1331-csharp/solutions/Solution110.cs
using _4MeOnWeb.SolutionAPI;

namespace _4MeOnWeb.Solutions {
    public class Solution110 : ImmutableSolution {
        public Solution110() :
            base("1.10", "Average Speed") {
        }

        public override IEnumerable<string> Answers {
            get {
                double speed = 14.0 / 1.6 / (45.0 / 60.0 + 30.0 / 60.0 / 60.0);

                return new[] {
                    $"Average speed is {speed:f6} km/h"
                    };
            }
        }
    }
}